azure/errors.go

```diff
@@ -0,0 +1,9 @@
+package azure
+
+const (
+	publishSettingsConfigurationError = "PublishSettingsFilePath is set. Consequently ManagementCertificatePath and SubscriptionId must not be set."
```

Comment:
Looks like publishSettingsConfigurationError and managementCertificateConfigurationError constants are specific for client.go file. And all of other file-specific constants are stored in their own files. 
I think we should use this file for reusable consts only, or move everything here.